### PR TITLE
Fixes #34117 - Drop ancient models:consolidate task

### PR DESCRIPTION
**Pull request by tbrisker**
This task was written 10 years ago and hasn't seen any updates since. It
is likely broken and useless.

### CODE REVIEW

File: lib/tasks/model.rake [hunk 1]
```diff
@@ -1,83 +0,0 @@
-# TRANSLATORS: do not translate
-desc <<~EOT
-  When Foreman imports a new host by reading its facts it will create the host and
-  set its model to a value based upon the facts. This can result in a multiplicity
-  of similarly named models. These may be condensed donw into a few basic models
```
Comment (ekohl) [inline lib/tasks/model.rake:5]: Did you find this when looking for typos? ;)

**Review by ekohl — APPROVED** [review]
I never knew we had this.

File: lib/tasks/model.rake (hunk 1 above)
Comment (tbrisker) [reply lib/tasks/model.rake:5]: Actually when looking for places that still check for `SETTINGS[:unattended]`, but i did also notice this

**Comment by lzap** [comment]
And the day feels better, thanks!

**Pull request merged**